### ISSUE TITLE
Suggestion: show that a change has been made

### ISSUE THREAD

**Issue by JulianBroudy** [autoprefixer/autoprefixer.github.io#57]
Hello,

I came across the [prefixer](https://autoprefixer.github.io/) while searching for an online generator that would "make sure" my CSS code would work on almost all browsers.

First of all, let me say VERY NICE, Kudos!

I'm not sure this is the place to do that, I sure hope it is.
I had an idea, for a newbie like me, who barely knows anything about CSS's history, the various browsers, and all the available syntaxes. It isn't immediately obvious that my code has been altered. Since I would also like to learn rather than just paste and copy the "new" corrected version, I would like to be able to paste different chunks of code and see what did I miss.
So I think some kind of boolean indicator would be nice, an icon that would "light up" to indicate that the pasted code has been "prefixed" somewhere so to speak.

For an extra challenge or a slightly more complicated enhancement maybe highlight the changes or the lines that were changed?

In case this is too much, maybe it would be possible to let the user enable that kind of option, especially considering many would paste huge chunks of code and that would be a waste of resources to highlight every line.

If that's something you might be interested in, I could, if possible, take responsibility for that and try to implement it? I haven't made any contributions yet and looking for my first one.

Looking forward to hearing from you.

**Comment by ymatuhin** [comment]
@JulianBroudy hey, first of all thanks for the ideas.

> So I think some kind of boolean indicator would be nice, an icon that would "light up" to indicate that the pasted code has been "prefixed" somewhere so to speak.

Are you talking about the moment when input = output?

> highlight the changes or the lines that were changed

That's an amazing idea. Should be very informative for everybody. 

So if you want to try to implement anything, you're very welcome. You can freely ask any questions right here and I'll do my best to help you.

**Comment by JulianBroudy** [comment]
> Are you talking about the moment when input = output?
> 
> >

Actually I meant the opposite.
Light up when a change has actually been made indicating that the output **should** be copied.
Although, I can see why indicating that nothing has changed might be a better idea.
I will go with whatever you think is best.

I will start digging as soon as I get a free moment as I am currently trying to keep up with my courses.

**@ymatuhin, so I took a quick look and I have 1 main question:**

**What is the difference between the main.js & the main.js in assets folder? are they both used?**
I'm sorry I'm just seeing some new technologies and languages so I'm having a little trouble understanding what's what or where do things begin, i.e., the workflow. 
Maybe you could point me to what should I read or look up in order to better understand the structure.

**Comment by pi7dets** [comment]
I agree, it would be useful. When inserting a large piece of CSS, I would like to see what has changed.
